generation_config.yaml

```diff
@@ -0,0 +1,1974 @@
+gapic_generator_version: 2.35.0
+protobuf_version: '25.2'
+googleapis_commitish: 20c357f6f5c7b995b617b15cab61999e6574bdab
+owlbot_cli_image: sha256:623647ee79ac605858d09e60c1382a716c125fb776f69301b72de1cd35d49409
+synthtool_commitish: 6612ab8f3afcd5e292aecd647f0fa68812c9f5b5
+template_excludes:
+- .github/*
+- .kokoro/*
+- samples/*
+- CODE_OF_CONDUCT.md
+- CONTRIBUTING.md
+- LICENSE
+- SECURITY.md
+- java.header
+- license-checks.xml
+- renovate.json
+- .gitignore
+libraries:
+- api_shortname: accessapproval
+  name_pretty: Access Approval
+  product_documentation: https://cloud.google.com/access-approval/docs/
+  api_description: enables controlling access to your organization's data by Google
+    personnel.
+  release_level: stable
+  GAPICs:
+  - proto_path: google/cloud/accessapproval/v1
+
+- api_shortname: accesscontextmanager
+  name_pretty: Identity Access Context Manager
+  product_documentation: n/a
```

Comment:
YAML has null.

Reply:
This is because `product_documentation` is `n/a`.

We should verify the correctness of a field for new libraries.